brainscore_language/artificial_subject.py

```diff
@@ -99,7 +99,12 @@ class RecordingTarget:
 
     class RecordingType:
         """ method of recording """
-        spikerate_exact = "spikerate_exact"  # the exact spike-rate activity of each neuron
+
+        spikerate_exact = "spikerate_exact"
```

Comment:
yes, that's a reasonable implementation of it, although in principle it's up to the model. I think I will remove this again though, because we don't have any single-neuron electrode recordings here so far